.github/workflows/linux-tests.yml

```diff
@@ -0,0 +1,125 @@
+name: GitHub Actions CI - (Linux)
+on:
+  pull_request:
+    paths-ignore:
```

Comment:
Is there any way of extracting this to a list that can be shared somewhere?

Reply:
With `paths-ignore` no. With another job running before tests and determining if they should be run, maybe, but then we would get the "skipped" spam.

Reply:
@dawidd6 Ok. I think this is fine for now but eventually I think the "skipped" would be preferable.